lib/modules/presets.js

```diff
@@ -47,36 +47,36 @@ module.options = {
 				'commentPreview',
 				'localDate',
 				'noParticipation',
-			], '*');
-		}),
+			], '*')
+		)),
 	},
 	cleanSlate: {
 		title: 'presetsCleanSlateTitle',
 		description: 'presetsCleanSlateDesc',
 		type: 'button',
 		text: 'apply preset',
-		callback: confirmPreset(() => {
-			toggleModules(undefined, '*');
-		}),
+		callback: confirmPreset(() => (
```

Comment:
`toggleModules` is a bit slow, so returning a promise prevents the window be closed prematurely.